scripts/data_files/driver_templates/psa_crypto_driver_wrappers.c.jinja

```diff
@@ -2332,4 +2332,29 @@ psa_status_t psa_driver_wrapper_mac_abort(
     }
 }
 
+struct psa_entropy_acc;
+
+psa_status_t psa_driver_collect_entropy( size_t sz, uint8_t *buf, struct psa_entropy_acc *acc )
+{
+    size_t estimate_bits = 0;
+    int ret = PSA_SUCCESS;
+
+{% if not entropydrivers %}
+    (void)sz;
+    (void)buf;
+    (void)acc;
+    (void)estimate_bits;
+{% endif %}
+{% for driver in entropydrivers %}
+    ret = $driver(sz, &estimate_bits, buf, MBEDTLS_ENTROPY_BLOCK_SIZE);
```

Comment:
What is `sz` supposed to be? It isn't in the draft specification. Instead, there is a `uint32_t flags` parameter. Mbed TLS doesn't need to support configurable flags at this stage, but it should implement the correct prototype, and I think it should pass `PSA_DRIVER_GET_ENTROPY_BLOCK` since Mbed TLS wants entropy and will try again anyway if it doesn't get it.

Reply:
That was a mistake. It should be PSA_DRIVER_GET_ENTROPY_BLOCK. Fixed.